### ISSUE TITLE
Data fails to persist on remote host.

### ISSUE THREAD

**Issue by mcfwesh** [kartoza/docker-geoserver#206]
In my localhost, my data persists, even when I restart or rebuild my containers. I can find my workspaces, stores and layers together with my settings. More importantly, I can login with my chosen username and password indicated in the `.env` file.
This is not the case for my remote host where I deploy. Since I only cloned my repo in the host, I expected the same as my local development. 
However, I could not even login with the same username and password indicated in the `.env` file I used in the local development I had to use the default one. As a result, I could not even see my layers, workspaces, and more importantly my datadir folder was no where to be found. Interestingly, when I remove/add plugins to the `env` file, and run the compose, it changes accordingly, which is ideal. Any pointers?
TL;DR
### On remote host:

- Cannot login with specified username and password as indicated in `.env` which results in the inability to see uploaded layers

- Cannot see `datadir` directory, after I used the default `admin` and `geoserver` login details

docker-compose file:
```
version: "3.4"
services:
  # back-end api built with golang
  geoserver:
    container_name: "geoserver"
    image: kartoza/geoserver:${GEOSERVER_VERSION_TAG}
    volumes:
      - "./data:/opt/geoserver/data_dir"
    ports:
      - 8080:8080
    restart: "always"
    environment:
      - GEOSERVER_DATA_DIR=${GEOSERVER_DATA_DIR}
      - GEOWEBCACHE_CACHE_DIR=${GEOWEBCACHE_CACHE_DIR}
      - GEOSERVER_ADMIN_PASSWORD=${GEOSERVER_ADMIN_PASSWORD}
      - GEOSERVER_ADMIN_USER=${GEOSERVER_ADMIN_USER}
      - STABLE_EXTENSIONS=${STABLE_EXTENSIONS}
      - COMMUNITY_EXTENSIONS=${COMMUNITY_EXTENSIONS}
      - INITIAL_MEMORY=${INITIAL_MEMORY}
      - MAXIMUM_MEMORY=${MAXIMUM_MEMORY}
  nginx:
    build:
      context: nginx
      target: dev
    ports:
      - "80:80"
      - "443:443"
    depends_on:
      - h2-atlas
      - geoserver

  h2-atlas:
    build:
      context: client
      target: dev
    volumes:
      - "./client:/root"
    ports:
      - "3000:3000"

```

**Comment by NyakudyaA** [comment]
@mcfwesh Will check this later

**Comment by mcfwesh** [comment]
Hello. Any solution yet? 

Apparently, when I tried earlier versions, the data persisted, but some useful extensions that I need could not be installed. This development has slowed me down

**Comment by NyakudyaA** [comment]
This is more a user problem than the image itself
